src/Account/AccountCollection.php

```diff
@@ -7,7 +7,7 @@
 /**
  * @author Milan Matějček
  */
-class Accounts
+class AccountCollection implements \Countable, \IteratorAggregate
```

Comment:
I can update it, it's not much problem. Maybe I can try to measure it, because I don't belive @dg would use it if it was slow :-)